### PR TITLE
[UXE-2855] fix: requires delivery protocol with https

### PR DESCRIPTION
**Pull request by pauloSF0**
### What is the new behavior introduced by this PR?
fix: requires delivery protocol with https

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.

https://github.com/aziontech/azion-console-kit/assets/110847590/61266b1c-2fbd-4c77-9080-61d44f23ba88

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by peterpaulo-azion — DISMISSED** [review]

File: src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue [hunk 1]
```diff
@@ -346,14 +354,19 @@
...
     }
 
     return options.map((option) => {
       if (option.requires) {
-        const requires = conditionsMap[option.value] || !props.isEnableApplicationAccelerator
+        let requires = conditionsMap[option.value]
 
+        if (requires === undefined) {
+          requires = !props.isEnableApplicationAccelerator
+        }
```
Comment (william-tome) [inline src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue:369]: Eu não entendi a alteração.

File: src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue [hunk 2]
```diff
@@ -78,6 +78,14 @@
     return ' - Requires Application Accelerator'
   })
 
+  const showLabelHttps = computed(() => {
+    if (props.isEnableApplicationAccelerator || props.isDeliveryProtocolHttps) {
+      return ''
+    }
+
+    return '- requires Delivery Protocol with HTTPS'
```
Comment (william-tome) [inline src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue:86]: o `requires` é minúsculo mesmo?

File: src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue (hunk 1 above)
Comment (pauloSF0) [reply src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue:369]: antes a validação era no OR ai não tava dando certo

File: src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue [hunk 3]
```diff
@@ -346,14 +360,17 @@
...
+      redirect_http_to_https: isDisabledHttps.value,
       optimize_images: !props.isImageOptimization
     }
 
     return options.map((option) => {
       if (option.requires) {
-        const requires = conditionsMap[option.value] || !props.isEnableApplicationAccelerator
+        let requires = conditionsMap[option.value]
 
+        if (requires === undefined) {
```
Comment (lucasmendes21) [inline src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue:371]: Acho que poderia usar if (!requires)

**Review by lucasmendes21 — DISMISSED** [review]
Aprovei mais deixei um comentário para melhoria.

**Review by peterpaulo-azion — APPROVED** [review]

**Review by lucasmendes21 — APPROVED** [review]

**Review by matheus-sunderhus — APPROVED** [review]
LGTM

**Pull request merged**